cmd/up/activate.go

```diff
@@ -457,3 +462,29 @@ func getPullingMessage(message, namespace string) string {
 	toReplace := fmt.Sprintf("%s/%s", registry, namespace)
 	return strings.Replace(message, toReplace, okteto.DevRegistry, 1)
 }
+
+// waitUntilAppIsAwaken waits until the app is awaken checking if the annotation dev.okteto.com/state-before-sleeping is present in the app resource
+func (up *upContext) waitUntilAppIsAwaken(ctx context.Context, app apps.App) error {
+	if _, ok := app.ObjectMeta().Annotations[model.StateBeforeSleepingAnnontation]; !ok {
+		return nil
+	}
+
+	timeout := 5 * time.Minute
```

Comment:
@pchico83 I'm setting 5 minutes as the maximum time to wait for a resource to be awaken. Do you think we should increase it?

Reply:
I think we should be using `dev.timeout.resources`. Default value is 120 seconds. I don't know if that works for you but if we use it it's already configurable in the dev section

Reply:
Not sure if 120 seconds would be enough. When things starts to be waken up in order it might take a bit more (taking also into account the time for the job to be scheduled)